CartPole/cartpole_model_tf.py

```diff
@@ -1,16 +1,20 @@
 from types import SimpleNamespace
-from typing import Union
-from CartPole.state_utilities import (
-    create_cartpole_state,
-    ANGLE_IDX, ANGLED_IDX, POSITION_IDX, POSITIOND_IDX, ANGLE_COS_IDX, ANGLE_SIN_IDX
-)
-import tensorflow as tf
 
+import numpy as np
+import tensorflow as tf
+import os
+import yaml
+from others.globals_and_utils import create_rng
+from others.p_globals import (J_fric, L, M, M_fric, TrackHalfLength,
+                              controlBias, controlDisturbance, g, k, m, u_max,
+                              v_max)
 from SI_Toolkit.TF.TF_Functions.Compile import Compile
 
-from others.p_globals import (
-    k, M, m, g, J_fric, M_fric, L, v_max, u_max, controlDisturbance, controlBias, TrackHalfLength
-)
+from CartPole.state_utilities import (ANGLE_COS_IDX, ANGLE_IDX, ANGLE_SIN_IDX,
+                                      ANGLED_IDX, POSITION_IDX, POSITIOND_IDX,
+                                      create_cartpole_state)
+
+config = yaml.load(open(os.path.join(os.path.dirname(__file__), "..", "config.yml"), "r"), Loader=yaml.FullLoader)
```

Comment:
Check

Reply:
Created a new helper `load_config(filename: str)` in `globals_and_utils` to perform the try/except config import where necessary. It has the advantage that if we want to later change the import precedence or places where to look for configs, then the change needs to be made only in one location